devito/core/cpu.py

```diff
@@ -147,7 +147,7 @@ def _specialize_iet(cls, graph, **kwargs):
         sregistry = kwargs['sregistry']
 
         # Distributed-memory parallelism
-        mpiize(graph, sregistry=sregistry, options=options)
+        mpiize(graph, **kwargs)
```

Comment:
@mloubout I decided you were right, we should just propagate the kwargs through the passes. However, I'll do this incrementally. I've started here...